pkgs/top-level/python-packages.nix

```diff
@@ -13940,6 +13940,23 @@ in {
     };
   });
 
+  ltc_scrypt = buildPythonPackage rec {
```

Comment:
That's because its relatively new and we're in the process of actually getting all the expressions there.